independent-publisher-connectors/Zoom Meetings/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Zoom Client ID",
+          "scopes": [
+            "meeting:read:admin meeting:read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://zoom.us/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://zoom.us/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://zoom.us/oauth/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Akuthota Deekshith"

```

Comment:
@A-Deekshith Please add `stackOwner` property as well.

Docs - [Adding Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)